tests/test_post_load.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+
+import marshmallow
+
+import marshmallow_dataclass
+
+
+class TestFieldForSchema(unittest.TestCase):
```

Comment:
It might be worth adding a test for `many=True`.

Reply:
done